pkg/operator/workloads/api_workload.go

```diff
@@ -299,16 +299,26 @@ func tfAPISpec(
 		},
```

Comment:
Do we need to update the ID calculation in `operator/context/apis.go`? While you have that file open, do you mind checking if we missed anything else we should add there?

Reply:
checked